src/ElasticsearchService.php

```diff
@@ -158,8 +162,10 @@ public function execute(Search $search)
      */
     public function count(Search $search): int
     {
+        $index = IndexNamePrefixer::getPrefixedIndexName($search->getIndex());
```

Comment:
@Jalle19 Can't we just make another method for `Search` likes `$search->getPrefixedIndexName()`?

Reply:
Yeah I suppose

Reply:
Actually now that I look at the file again, IndexNamePrefixer is used in a lot of other methods here so I don't think it makes sense to spread it to yet another class. It would be best to try to contain the prefixing logic, but unfortunately there's no single place where it can be done.

Reply:
Okay